S/Spot_julia/build_tarballs.jl

```diff
@@ -0,0 +1,102 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Spot_julia"
+version = v"2.9.7"
+julia_version = v"1.6.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://www.lrde.epita.fr/dload/spot/spot-2.9.7.tar.gz","1eea67e3446cdbbbb705ee6e26fd869020cdb7d82c563fead9cb4394b9baa04c"),
+    GitSource("https://github.com/MaximeBouton/spot_julia.git", "1357250568e0ecf6fdbe854460ac88de17f9345e")
+    ]
+    
+    # Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd spot-2.9.7/
+# make sure pdf is older than .tex to not trigger latexmk during build 
+# touch doc/tl/tl.tex
+# touch doc/tl/tl.pdf
+./configure --prefix=${prefix}/spot-build --build=${MACHTYPE} --host=${target} --disable-python
+make -j${nproc} && make install
+
+# build with cmake 
+cd $WORKSPACE/srcdir/spot_julia/spot_julia
+# edit the CMAKE script to find libcxxwrap 
+# "/home/maxime/cxxwrap-test/libcxxwrap-julia-build"
+
+# edit the CMAKE script to find spot-build
+sed -i 's#\${CMAKE_SOURCE_DIR}/../spot-build/#\${CMAKE_INSTALL_PREFIX}/spot-build/#g' CMakeLists.txt
+
+if [[ $target == *"mingw"* ]]; then
+  cp -L $WORKSPACE/destdir/spot-build/bin/*.dll $WORKSPACE/destdir/spot-build/lib/
+  rm $WORKSPACE/destdir/spot-build/bin/*.dll
+  windows_extra_flags="-DCMAKE_FIND_LIBRARY_SUFFIXES=.dll"
+fi
+
+# Override compiler ID to silence the horrible "No features found" cmake error
+if [[ $target == *"apple-darwin"* ]]; then
+  macos_extra_flags="-DCMAKE_CXX_COMPILER_ID=AppleClang -DCMAKE_CXX_COMPILER_VERSION=10.0.0 -DCMAKE_CXX_STANDARD_COMPUTED_DEFAULT=11"
+fi
+Julia_PREFIX=$prefix
+mkdir build
+cd build
+cmake -DJulia_PREFIX=$Julia_PREFIX -DCMAKE_FIND_ROOT_PATH=$prefix -DJlCxx_DIR=$prefix/lib/cmake/JlCxx -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} $macos_extra_flags $windows_extra_flags -DCMAKE_BUILD_TYPE=Release ../
+VERBOSE=ON cmake --build . --config Release --target install -- -j${nproc}
+
+install_license $WORKSPACE/srcdir/spot_julia/LICENSE.md
+
+exit
```

Comment:
I removed the useless comments and made the overly long line easier to read by splitting it
```suggestion
cd $WORKSPACE/srcdir/spot-2.9.7/
./configure --prefix=${prefix}/spot-build --build=${MACHTYPE} --host=${target} --disable-python
make -j${nproc}
make install

# build with cmake 
cd $WORKSPACE/srcdir/spot_julia/spot_julia

# edit the CMAKE script to find spot-build
sed -i 's#\${CMAKE_SOURCE_DIR}/../spot-build/#\${CMAKE_INSTALL_PREFIX}/spot-build/#g' CMakeLists.txt

if [[ $target == *"mingw"* ]]; then
  cp -L ${prefix}/spot-build/bin/*.dll ${prefix}/spot-build/lib/
  rm ${prefix}/spot-build/bin/*.dll
  windows_extra_flags="-DCMAKE_FIND_LIBRARY_SUFFIXES=.dll"
fi

# Override compiler ID to silence the horrible "No features found" cmake error
if [[ $target == *"apple-darwin"* ]]; then
  macos_extra_flags="-DCMAKE_CXX_COMPILER_ID=AppleClang -DCMAKE_CXX_COMPILER_VERSION=10.0.0 -DCMAKE_CXX_STANDARD_COMPUTED_DEFAULT=11"
fi
Julia_PREFIX=$prefix
mkdir build
cd build
cmake -DJulia_PREFIX=$Julia_PREFIX \
    -DCMAKE_FIND_ROOT_PATH=$prefix \
    -DJlCxx_DIR=${libdir}/cmake/JlCxx \
    -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    ${macos_extra_flags} \
    ${windows_extra_flags} \
    -DCMAKE_BUILD_TYPE=Release \
    ..
VERBOSE=ON cmake --build . --config Release --target install -- -j${nproc}
install_license $WORKSPACE/srcdir/spot_julia/LICENSE.md
```